fladle-plugin/src/main/java/com/osacky/flank/gradle/YamlConfigWriterTask.kt

```diff
@@ -16,10 +18,11 @@ import javax.inject.Inject
 @DisableCachingByDefault(because = "The task writes a small file from in memory properties and does not benefit from caching.")
 open class YamlConfigWriterTask @Inject constructor(
   @get:Nested val base: FlankGradleExtension,
-  @get:Nested val config: FladleConfig,
+  config: FladleConfig,
```

Comment:
I think this means that changes to this configuration object would not be picked up.

Can you clarify what the benefit of configuring it this way is? I'm not understanding the scenario here.